core/src/main/scala/kafka/zk/ZkData.scala

```diff
@@ -58,6 +58,30 @@ object BrokerIdsZNode {
   def encode: Array[Byte] = null
 }
 
+class BrokerInfo(id: Int,
```

Comment:
To be consistent, perhaps this should be name BrokerIdZNode.

Reply:
There's already a BrokerIdZNode in ZKData that BrokerInfo calls. It stores broker id, listeners and other info rather than being passed as args later to KafkaZkClient and consequently KafkaController.